### ISSUE TITLE
Screen Readers Cannot Interact with "View the autosave" Notice

### ISSUE THREAD

**Issue by donalirl** [Automattic/wp-calypso#41784]
#### Steps to reproduce

1. Make an edit to a page or post and reload the page without saving. The goal is to trigger the notice "There is an autosave of this post that is more recent than the version below. View the autosave"
2. Attempt to interact with this notice using a screen reader

#### What I expected

I expected to be able to swipe left to right with a screen reader through the options this notice gives a user. I should be able to tap to open the autosave, or swipe over to the dismiss button then tap to dismiss the notice.

#### What happened instead

Either nothing happens or the page freezes.

#### Screenshot

https://d.pr/i/XnjhEA

#### Browser / OS version

**Desktop**
Mac Catalina  with Safari 13 using Apple Voiceover

**Smartphone**
 - Device: iPod Touch
 - Browser: Safari Latest
 - Screen reader: Apple Voiceover

#### Context / Source

#p4vt7e-6n-p2

#journey #empathy #livesharing #accessibility #painpoint #user-report

**Comment by sixhours** [comment]
I tested this on my iPhone and desktop with Voiceover and can confirm the draft/autosave is loaded properly and can be accessed by tapping/tabbing depending on the device:

https://user-images.githubusercontent.com/2124984/128395521-3dc6bc50-8f37-4074-9a37-321b450500c9.mov

That said, this appears to be a slightly different notice from the one mentioned in the original ticket. This is a local browser storage backup notice. I'm not sure how to trigger the "View the autosave" notice here.

**Comment by sixhours** [comment]
Aha, figured out how to trigger the autosave notice; make a change and immediately Preview the change, then reload without saving.

I'm still able to navigate to the autosave link, both on desktop and my iPhone, and dismiss the notice in either case:

https://user-images.githubusercontent.com/2124984/128398567-88e6e009-c56d-4014-ab53-cc1ebd6b65fc.mov

I've browsed through the closed PRs in Gutenberg and the commit history to the file where that message is located to see if I could find the one. Nothing jumps out at me, although the main component was completely refactored back in December: https://github.com/WordPress/gutenberg/commit/225b075bdf8099efa06f0e750439c72a8a6d9e4e#diff-d364bf913a3b8f8eb8f79e2b52c75bd4130248c5a7c94fe021dc39602b19b697 and there's been work done around focusing in the editor: https://github.com/WordPress/gutenberg/pull/28191

Short of testing past iterations of Gutenberg with `git bisect` or something, I'm going to say this was fixed at some point in the last year. Feel free to reopen if there's something I've missed!
